packages/truffle-contract-schema/index.js

```diff
@@ -41,6 +41,8 @@ var properties = {
         } catch (e) {
           value = undefined;
         }
+      } else if (typeof value === "object") {
+        value.forEach(item => delete item.signature);
```

Comment:
I was debating whether we can allow this mutation, but I don't see any other prior precedent of one of these `transform` functions mutating an object. I think it's probably a risky move, sorry. Here I'll take a stab, how's this:
```suggestion
        value = Object.entries(value)
          .filter(([key]) => key !== "signature")
          .reduce((a, b) => ({ ...a, ...b }), {});
```

Reply:
The other thought I have here is whether or not we should make this an `else`. Should we just treat it as a prerequisite that a string value here conform?

Reply:
Oh shoot my code is wrong. One sec.

Reply:
Ah, should have clarified in the code. We only want `abi`'s (array typed) to be returned w/o function signatures (if they exist). Pushed something a little more explicit that retains immutability. 🤞

Reply:
Oh I forgot the outer loop! I like your new solution a lot more anyway